tests/unit/lms/resources/_js_config_test.py

```diff
@@ -155,8 +162,13 @@ def bearer_token_schema(BearerTokenSchema):
 
 
 @pytest.fixture
-def config(context, pyramid_request):
-    return JSConfig(context, pyramid_request).config
+def js_config(context, pyramid_request):
+    return JSConfig(context, pyramid_request)
```

Comment:
Sometimes it's convenient for tests to receive the `JSConfig.config` dict as a fixture directly, sometimes they need the containing `JSConfig` object itself. So I've made two separate fixtures for them